Resources/DNNCE_pt-BR.dnn

```diff
@@ -0,0 +1,2291 @@
+						</languageFile>
+						<languageFile>
+							<path>DesktopModules\Admin\ViewProfile\App_LocalResources</path>
+							<name>Settings.ascx.pt-BR.resx</name>
+						</languageFile>
+						<languageFile>
+							<path>DesktopModules\Admin\ViewProfile\App_LocalResources</path>
+							<name>SharedResources.pt-BR.resx</name>
+						</languageFile>
+						<languageFile>
+							<path>DesktopModules\Admin\ViewProfile\App_LocalResources</path>
+							<name>ViewProfile.ascx.pt-BR.resx</name>
+						</languageFile>
+						<languageFile>
+							<path>Install\App_LocalResources</path>
+							<name>Installwizard.aspx.pt-BR.resx</name>
+						</languageFile>
+						<languageFile>
+							<path>Install\App_LocalResources</path>
+							<name>UpgradeWizard.aspx.pt-BR.resx</name>
+						</languageFile>
+						<languageFile>
+							<path>Portals\_default</path>
+							<name>Blank Website.template.pt-BR.resx</name>
+						</languageFile>
+						<languageFile>
+							<path>Portals\_default</path>
+							<name>Default Website.template.pt-BR.resx</name>
+						</languageFile>
+					</languageFiles>
+				</component>
+			</components>
+		</package>
+		<package name="DDRMenu_pt-BR" type="ExtensionLanguagePack" version="00.00.00">
+			<friendlyName>DDR Menu Português (Brasil, alfabetização internacional)</friendlyName>
+			<description><![CDATA[Portuguese (Brazil, International Sort) language pack for DDR Menu.<br />This translation of DNN into Portuguese (Brazil, International Sort) was done by <a href="http://www.disgrafic.com">Disgrafic ITec SL</a>.]]></description>
+			<iconFile />
+			<owner>
+				<name>DNN Platform</name>
+				<organization>.NET Foundation</organization>
+				<url>https://www.dnncommunity.org</url>
+				<email>info@dnncommunity.org</email>
+			</owner>
+			<license><![CDATA[DNN - http://www.dnnsoftware.com<br>Copyright (c) 2002-2017<br>by DNN Corp.<br><br>Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:<br><br>The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.<br><br>THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.]]></license>			
```

Comment:
There are multiple instances of this wrong license across this file, it's not easy to suggest them all in the browser though. If you can handle this change in the whole file, it would be awesome.